### PR TITLE
Migrate GrpcClientPipeline to use toolkit

### PR DESCRIPTION
**Pull request by shinfan**
Toolkit PR: https://github.com/googleapis/toolkit/pull/1159

### CODE REVIEW

File: pipeline/tasks/package_metadata_tasks.py [hunk 1]
```diff
@@ -39,6 +39,8 @@ def execute(self, api_name, api_version, organization_name, output_dir,
         with open(package_defaults_yaml) as dep_file:
             package_defaults = yaml.load(dep_file)
 
+        package_type = api_name == 'common-protos' and 'grpc_common' or 'grpc_client'
```
Comment (garrettjonesgoogle) [inline pipeline/tasks/package_metadata_tasks.py]: this doesn't really make sense to me.

Comment (shinfan) [inline pipeline/tasks/package_metadata_tasks.py]: I can add a `package_type` field in the artman config.

Comment (garrettjonesgoogle) [inline pipeline/tasks/package_metadata_tasks.py]: oh I think I finally figured it out, it's a fancy tertiary operation. 

In any case, I much prefer this to be in the artman config.

Comment (shinfan) [inline pipeline/tasks/package_metadata_tasks.py]: Done

**Comment by shinfan** [comment]
PTAL

**Comment by garrettjonesgoogle** [comment]
LGTM

**Pull request merged**